.github/workflows/packages.yaml

```diff
@@ -65,3 +69,33 @@ jobs:
       with:
         name: "${{ steps.generate-artifact-properties.outputs.artifact-name }}"
         path: 'packages'
+  BuildMarinerRelease:
+    runs-on: 'ubuntu-18.04'
+    steps:
+    - uses: 'actions/checkout@v1'
+      with:
+        submodules: 'recursive'
+    - name: 'Run'
+      run: |
+        docker run --rm \
+          -v "$GITHUB_WORKSPACE:/src" \
+          -e "ARCH=amd64" \
+          -e "PACKAGE_VERSION=$PACKAGE_VERSION" \
+          -e "PACKAGE_RELEASE=$PACKAGE_RELEASE" \
+          --privileged \
+          "ubuntu:18.04" \
+          '/src/ci/package-mariner.sh'
+      env:
+        ARCH: "amd64"
+        PACKAGE_VERSION: "${{ env.PACKAGE_VERSION }}"
+        PACKAGE_RELEASE: "${{ env.PACKAGE_RELEASE }}"
+    - name: 'Generate artifact properties'
+      id: 'generate-artifact-properties'
+      run: |
+        echo "::set-output name=artifact-name::packages_mariner_amd64"
+    - name: 'Upload'
+      uses: 'actions/upload-artifact@v1'
+      with:
+        name: "${{ steps.generate-artifact-properties.outputs.artifact-name }}"
+        path: 'packages'
+  
```

Comment:
Remove this trailing whitespace.

Reply:
trailing space should be removed now

Reply:
You've also deleted the newline at the end that shouldn't have been deleted. (Notice the diff in the GitHub UI.)

Reply:
fixed line ending